src/database/seeds/0002-update-canonical_fields.sql

```diff
@@ -0,0 +1,188 @@
+('Proposal Context', 'proposal_context', '{ "type": "string" }' ),
+('Proposal Leadership', 'proposal_leadership', '{ "type": "string" }' ),
+('Proposal Learning and Evaluation', 'proposal_learning_and_evalution', '{ "type": "string" }' ),
+('Proposal Past Performance', 'proposal_past_performance', '{ "type": "string" }' ),
+('Steps to Prevent Illegal Activity', 'steps_to_prevent_illegal_activity', '{ "type": "string" }' ),
+('Proposal Funds Support Unsupervised Entities', 'proposal_funds_support_unsupervised_entities', '{ "type": "boolean" }' ),
+('Proposal Funds Support Scholarships or Awards', 'proposal_funds_support_scholarships_or_awards', '{ "type": "boolean" }' ),
+('Proposal Funds Support Scholarships or Awards Explanation', 'proposal_funds_support_scholarships_or_awards_explanation', '{ "type": "string" }' ),
+('Proposal Payments to Non-Employees', 'proposal_payments_to_non_employees', '{ "type": "boolean" }' ),
+('Proposal Payments to Non-Employees Explanation', 'proposal_payments_to_non_employees_explanation', '{ "type": "string" }' ),
+('Proposal Location of Work 2', 'proposal_location_of_work_2', '{ "type": "string" }' ),
+('Proposal Funds Locations Under US Sanction', 'proposal_funds_locations_under_us_sanction', '{ "type": "boolean" }' ),
+('Proposal Funds Locations Under US Sanction Explanation', 'proposal_funds_locations_under_us_sanction_explanation', '{ "type": "string" }' ),
+('Proposal Supports Research', 'proposal_supports_research', '{ "type": "boolean" }' ),
+('Proposal Involves Desk Research', 'proposal_involves_desk_research', '{ "type": "boolean" }' ),
+('Proposal Collects New Data from Individuals', 'proposal_collects_new_data_from_individuals', '{ "type": "boolean" }' ),
+('Proposal Collects Personal Data', 'proposal_collects_personal_data', '{ "type": "boolean" }' ),
+('Proposal Collects New or Personal Data Explanation', 'proposal_collects_new_or_personal_data_explanation', '{ "type": "string" }' ),
+('Proposal Research Will be Published', 'proposal_research_will_be_published', '{ "type": "boolean" }' ),
+('Organization has Human Subjects Research Policy', 'organization_has_human_subjects_research_policy', '{ "type": "boolean" }' ),
+('Proposal Research Involves Minors', 'proposal_research_involves_minors', '{ "type": "boolean" }' ),
+('Organization has Child Protection Policy', 'organization_has_child_protection_policy', '{ "type": "boolean" }' ),
+('Proposal Funds Fiscal Agent Duties Research or Conference', 'proposal_funds_fiscal_agent_duties_research_or_conference', '{ "type": "boolean" }' ),
+('Proposal Funds Fiscal Agent Duties Research or Conference Explanation', 'proposal_funds_fiscal_agent_duties_research_or_conference_explanation', '{ "type": "string" }' ),
+('Additional Information From Applicant', 'additional_information_from_applicant', '{ "type": "string" }' ),
+('Organization Articles of Incorporation Document', 'organization_articles_of_incorporation_document', '{ "type": "uri" }' ),
```

Comment:
Hmm, I don't see that JSON Schema string type available at https://json-schema.org/draft/2020-12/json-schema-validation.html#name-resource-identifiers, I only see `uri`, `uri-reference`, `iri`, `iri-reference`, and `uuid` in that section. I don't see `url` anywhere on the page. But I suppose the JSON Schema here is incorrect since `uri` is supposed to apply to a `string`? Corrected it would be more like this, right?
```json
{ "type": "string", "format": "uri" }
```

Reply:
I found a good-enough way to deal with this with JSON Schema. I started with this:
```json
{ "type": "string", "allOf": [  { "format": "uri" }, { "pattern": "^https?://.*" } ] }
```
And eventually simplified down to this:
```json
{ "type": "string", "format": "uri", "pattern": "^https?://" }
```

Reply:
Aha, I missed that this was being fed directly into JSON Schema! Yes, that simplification looks good - thanks, @bickelj!

Reply:
While I think we have decided to ditch JSON Schema and make our own Domain-Specific Language for the schema, this change is what happened back then and still should be merged.